packages/discord.js/typings/index.test-d.ts

```diff
@@ -3034,14 +3028,11 @@ await guildScheduledEventManager.edit(snowflake, { recurrenceRule: null });
   });
 }
 
-await textChannel.send({
-  files: [
-    new AttachmentBuilder('https://example.com/voice-message.ogg')
-      .setDuration(2)
-      .setWaveform('AFUqPDw3Eg2hh4+gopOYj4xthU4='),
-  ],
-  flags: MessageFlags.IsVoiceMessage,
-});
+await textChannel.send(
```

Comment:
Shouldn't we keep a test for sending a voice message instead of replacing this with a plain text file?

Reply:
yea that was an accident, ty

> If possible, updating one of the runtime test files would be appreciated

are those actually.. maintained at all?